sdk/communication/communication-sms/review/communication-sms.api.md

```diff
@@ -10,13 +10,67 @@ import type { KeyCredential } from '@azure/core-auth';
 import type { OperationOptions } from '@azure/core-client';
 import type { TokenCredential } from '@azure/core-auth';
 
+// @public (undocumented)
+export interface AddOptions extends OperationOptions {
+}
+
+// @public (undocumented)
+export interface CheckOptions extends OperationOptions {
+}
+
+// @public (undocumented)
+export interface OptOutAddResult {
+    // (undocumented)
+    errorMessage?: string;
+    // (undocumented)
+    httpStatusCode: number;
+    // (undocumented)
+    to: string;
+}
+
+// @public (undocumented)
+export interface OptOutCheckResult {
+    // (undocumented)
+    errorMessage?: string;
+    // (undocumented)
+    httpStatusCode: number;
+    // (undocumented)
+    isOptedOut: boolean;
+    // (undocumented)
+    to: string;
+}
+
+// @public (undocumented)
+export interface OptOutRemoveResult {
+    // (undocumented)
+    errorMessage?: string;
+    // (undocumented)
+    httpStatusCode: number;
+    // (undocumented)
+    to: string;
+}
+
+// @public (undocumented)
```

Comment:
Could you add docstrings to all public bits of the API?

Reply:
Comments added